apis/duck/util/util_test.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+
+	"encoding/json"
+
+	duckv1alpha1 "github.com/knative/pkg/apis/duck/v1alpha1"
```

Comment:
I'm not sure I follow. I couldn't put it in ../ because of go circular dependency import problem. This depends on the v1alpha1 types and it depends on ../